mlx/primitives.cpp

```diff
@@ -402,6 +402,37 @@ std::pair<std::vector<array>, std::vector<int>> ArcTan::vmap(
   return {{arctan(inputs[0], stream())}, axes};
 }
 
+std::vector<array> ArcTan2::vjp(
+    const std::vector<array>& primals,
+    const std::vector<array>& cotangents,
+    const std::vector<int>& argnums,
+    const std::vector<array>&) {
+  return jvp(primals, cotangents, argnums);
+}
+
+std::vector<array> ArcTan2::jvp(
+    const std::vector<array>& primals,
+    const std::vector<array>& tangents,
+    const std::vector<int>& argnums) {
+  assert(primals.size() == 2);
+  assert(argnums.size() == 2);
+  array one = array(1., primals[0].dtype());
```

Comment:
Unused?

Reply:
I wasn't very sure--I added it since `jvp` and `vjp` are `virtual` in `Primitive`

Reply:
I meant the `array one` not the function itself which is should be added.

Reply:
Ooh, yes, nice catch, thanks!